From 632282145ba1d617f480cde74eef938826fef4d5 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Thu, 16 Jan 2025 11:24:12 -0800
Subject: Port better to NVHPC

See <https://savannah.gnu.org/support/?111172>.
* lib/autoconf/fortran.m4 (_AC_FC_LIBRARY_LDFLAGS):
Ignore -l options with embedded '='.
---
 lib/autoconf/fortran.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/autoconf/fortran.m4 b/lib/autoconf/fortran.m4
index 0fc7545d..4c39607c 100644
--- a/lib/autoconf/fortran.m4
+++ b/lib/autoconf/fortran.m4
@@ -664,7 +664,7 @@ while test $[@%:@] != 1; do
 	  ;;
 	  # Ignore these flags.
 	-lang* | -lcrt*.o | -lc | -lgcc* | -lSystem | -libmil | -little \
-	  |-LANG:=* | -LIST:* | -LNO:* | -link)
+	  | -[[lLR]]*=* | -LIST:* | -LNO:* | -link)
 	  ;;
 	-lkernel32)
 	  # Ignore this library only on Windows-like systems.
-- 
cgit v1.2.3-70-g09d2


